package.json

```diff
@@ -2,26 +2,28 @@
   "name": "kyber_dao_sc",
   "version": "2.0.0",
   "description": "This repository has the contracts for the KyberDao",
-  "main": "truffle.js",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/KyberNetwork/kyber_dao_sc.git"
+  "main": "hardhat.config.js",
+  "directories": {
+    "test": "test"
   },
   "scripts": {
     "test": "./test.sh",
     "t": "./test.sh",
-    "compile": "./compile.sh",
-    "c": "./compile.sh",
+    "compile": "printf 'yes' | ./compile.sh",
```

Comment:
Can remove the `printf 'yes'` for compile, cos only `yarn c` is used in the travis script. Small thing, leave it if u want

Reply:
Kk